### DISCUSSION TITLE
Kartoffel und Rüben ernter fahren Rückwärts

### DISCUSSION THREAD

**Discussion by Djannngooo** [Courseplay/Courseplay_FS22#957] [General]
Guten Tag, ich habe heuete folgendes festgestellt.
Rüben Missionen und Eigene felder mit Zuckerrüben, fährt der Rübenernter sporadisch rückwärts.
Ich weis nicht ob es an den einstellungen liegt oder ein Bug ist, aber es ist mir vorher nicht aufgefallen.
 Mal fährt er 4 bis 5 reihen und schwups, fährt er nach einer reihe einfach rückwärts und hält aber auch nicht an. Er dreht einfach vorne nicht um. Spiele auf der Marsch und es ist bei jeden Ingame ernter und bei den Mods ernter leider auch.

Vielleicht kennt jemand das Problem.

**Comment by Tensuko** [comment]
Mehr Vorgewende, skip row, turn on field off, etc.

**Reply by Djannngooo** [reply]
Okay, Was meinst du mit Feld ausschalten ?

**Reply by Tensuko** [reply]
Auf dem Feld Wenden ausschalten.

**Reply by Djannngooo** [reply]
okay, danke. Teste ich mal.
